fiatconnect-api.md

```diff
@@ -1890,51 +1894,88 @@ the UNIX timestamp that the request was sent. The second will be of the form `s=
 MUST be computed from the shared webhook private key and a dot-separated string consisting of the UNIX timestamp joined with the request body.
 This signature verification design is based off of [Persona's webhook documentation](https://docs.withpersona.com/docs/best-practices).
 
-# 6. AML Considerations
+# 6. Amounts
+
+In the interest of preventing rounding errors, [floating point errors](https://floating-point-gui.de/), or similar errors, numerical amounts 
+MUST be communicated as strings for every request parameter and every response field throughout the FiatConnect API. 
+
+Clients and servers alike SHOULD NOT cast fiat or crypto amounts to floating point numbers or integers, but should instead
+consider libraries such as [bignumber.js](https://github.com/MikeMcl/bignumber.js/) for handling large numbers and decimals 
+to an arbitrary number of decimal places.
+
+## 6.1 Units
+
+Except where otherwise noted, amounts of cryptocurrencies MUST be given in the units of that currency, e.g. `1.5` Celo means
```

Comment:
Perhaps add a note here specifying that the precision given for cryptocurrency amounts MUST NOT exceed the maximum possible precision for the given currency (i.e., that specified by its value for ERC20's [decimals](https://docs.openzeppelin.com/contracts/3.x/erc20#a-note-on-decimals))